assignments/mgmt-groups/mg-dts002/assign.appgateway_diagnostics.json

```diff
@@ -28,6 +28,6 @@
         "tier": "Free"
     },
     "type": "Microsoft.Authorization/policyAssignments",
-    "id": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyAssignments/HMCTSAppGWDiagnostic",
-    "name": "HMCTSAppGWDiagnostic"
+    "id": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyAssignments/HMCTSAGWDiagnosticGlobal",
+    "name": "HMCTSAGWDiagnosticGlobal"
```

Comment:
is it renamed because of a length limitation or as a better name?

Reply:
just bringing it in line with the way the others are named